build_scripts/build_usd.py

```diff
@@ -1123,6 +1158,47 @@ def _RunBuild(arch):
         CopyDirectory(context, "include/serial", "include/serial")
         CopyDirectory(context, "include/tbb", "include/tbb")
 
+        # generate TBBConfig.cmake so we have it available for
+        # the OpenUSD build - this also patches the generated
+        # configuration to adhere to the desired directory structure
+        # as well as adds TBB_DEBUG to the INTERFACE_INCLUDE_DEFINITIONS
+        cmakeInstallTBBCmd = 'cmake -DTBB_ROOT={root} -DTBB_OS=Darwin -P cmake/tbb_config_generator.cmake'.format(
+            root=context.instDir)
+        Run(cmakeInstallTBBCmd)
+        CopyFiles(context, os.path.join(context.instDir, "cmake", "TBBConfig.cmake"),
+                  os.path.join(context.instDir, "lib", "cmake", "TBB"))
+        CopyFiles(context, os.path.join(context.instDir, "cmake", "TBBConfigVersion.cmake"),
+                  os.path.join(context.instDir, "lib", "cmake", "TBB"))
+        
+        PatchFile(os.path.join(context.instDir, "lib/cmake/TBB/TBBConfig.cmake"),
```

Comment:
nitpick but the path components should be a list of strings instead of a single string since you're already using `os.path.join`

Reply:
This one makes sense to me, it's a different case than above because there wasn't already code there to mirror, will change.